job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -173,6 +179,22 @@ String kogitoWithSpecDroolsNightlyStage(String kogitoVersion, String kogitoBranc
     """
 }
 
+String serverlessLogicNightlyStage(String productVersion, String branch, String configBranch, String nexusSuffix = getNexusFromVersion(productVersion)) {
+    return """
+        stage('trigger KOGITO nightly job ${productVersion}') {
+            steps {
+                build job: 'kogito.nightly/${branch}', propagate: false, wait: true, parameters: [
```

Comment:
should we rename the nightly job to `serverless-logic.nightly` ?

Reply:
i would still keep there kogito, so kogito-swf.nightly?

Reply:
I would too, do you agree @radtriste

Reply:
ok